### PR TITLE
Kan 3 contest creation page UI

### PR DESCRIPTION
**Pull request by Yura-Kost**
In this branch i was working with page for creating post constest request.

Here i added some extra svgs, created flexible combobox component, modified scss files for input files to use webkit for applying custom design, and created the design itself

### CODE REVIEW

File: src/shared/ui/mainInfoRadioEl/ui/mainInfoRadioEl.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    text,
+    currSelected,
+    setCurrSelected,
+}: MainInfoRadioElProps) => {
+    const classname =
+        text === currSelected
+            ? 'mainInfoRadioEl_container active'
+            : 'mainInfoRadioEl_container'
+    return (
+        /* eslint-disable jsx-a11y/click-events-have-key-events */
```
Comment (VolodymyrKholiavko) [inline src/shared/ui/mainInfoRadioEl/ui/mainInfoRadioEl.tsx:25]: comments should be removed or resolved

File: src/widgets/galleryUpload/ui/galleryUpload.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                    <Icon Svg={questionMark} height={20} width={21} />
+                </HStack>
+                <Text Tag='p' className='header_label'>
+                    You can upload to this gallery examples of robots to be
+                    performed by competition participants, as well as
+                    photographs of prizes
+                </Text>
+            </VStack>
+
+            <div className='grid_photo_container'>
```
Comment (VolodymyrKholiavko) [inline src/widgets/galleryUpload/ui/galleryUpload.tsx:26]: should be mapped

File: src/widgets/prizeInformation/ui/prizeInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import './prizeInformation.scss'
+
+export const PrizeInformation = () => {
+    return (
+        <VStack className='prizeInformation_container'>
+            <Text Tag='h2' className='prizeInformation_header'>
+                Prize Information
+            </Text>
+            <VStack className='prizePlaces_container'>
+                <PrizePlace winIcon={win1} deleteIcon={trash} />
```
Comment (VolodymyrKholiavko) [inline src/widgets/prizeInformation/ui/prizeInformation.tsx:20]: should be mapped

**Review by VolodymyrKholiavko — CHANGES_REQUESTED** [review]
Good at all but still much incorrect architecture usages

**Review by VolodymyrKholiavko — CHANGES_REQUESTED** [review]
Done

File: src/shared/ui/mainInfoRadioEl/ui/mainInfoRadioEl.tsx (hunk 1 above)
Comment (FreadSt) [reply src/shared/ui/mainInfoRadioEl/ui/mainInfoRadioEl.tsx:25]: by the way, "resolve" means dealing with the warnings eslint

File: src/shared/ui/mainInfoRadioElContainer/ui/mainInfoRadioElContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+export const MainInfoRadioElContainer = ({
+    text,
+    svg,
+    children,
+    currActive,
+}: MainInfoRadioElContainerProps) => {
+    const [currSelected, setCurrSelected] = useState(currActive)
+    return (
+        <VStack className='mainInfoRadioElContainer_container'>
```
Comment (FreadSt) [inline src/shared/ui/mainInfoRadioElContainer/ui/mainInfoRadioElContainer.tsx:23]: should be renamed classname to shorter and understandable

File: src/widgets/mainInformation/ui/mainInformation.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+import { MainInfoRadioEl } from 'shared/ui/mainInfoRadioEl'
+import { MainInfoRadioElContainer } from 'shared/ui/mainInfoRadioElContainer'
+import { MainInformationCombobox } from 'shared/ui/mainInformationCombobox/ui/mainInformationCombobox'
+import { HStack, VStack } from 'shared/ui/stack'
+import { Text } from 'shared/ui/text'
+
+import './mainInformation.scss'
+
+
+const categories = [
```
Comment (FreadSt) [inline src/widgets/mainInformation/ui/mainInformation.tsx:21]: mock constants should be moved to separate directories

**Review by FreadSt — CHANGES_REQUESTED** [review]
Done. Good at all but need more attention to architecture and naming

**Review by FreadSt — APPROVED** [review]

**Review by FreadSt — CHANGES_REQUESTED** [review]
done

**Pull request merged**